platform-service-impl/src/main/java/esa/mo/platform/impl/provider/opssat/PowerControlOPSSATAdapter.java

```diff
@@ -123,17 +142,51 @@ public void enableDevices(final DeviceList inputList) throws IOException
       }
       if (payloadId != null) {
         switchDevice(payloadId, device.getEnabled());
+        if(!device.getEnabled())
```

Comment:
This is the part I was mentioning in the last issue#352 comment:
"So far my idea for OPSSAT was that when enableDevices() is called on PowerControl we check wether it is setting in on or off. If it is setting the device OFF, we mark the device status OFF right away. But when it is setting it ON, we will wait for the parameter read to tell us when it is ON and only then mark it available."

Would this be acceptable behaviour? So we do not depend on the timing (eg 60seconds).

Reply:
It would be acceptable for *most* of payloads, except for iADCS which gets a little more complicated... in principle we should not attempt to communicate with iADCS within the first 60 seconds of its power-channel coming online, because it is very sensitive to any premature communication attempts.

Reply:
See the code that follows. I have added a delay to ADCS activation.